<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>EmpPendingTable</class>
    <widget class="QDialog" name="EmpPendingTable">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>900</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Leave Applications</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="text">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Leave Applications&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignmentFlag::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="filterFrame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QLabel" name="filterLabel">
                                <property name="text">
                                    <string>Filter Status:</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QRadioButton" name="allRadioButton">
                                <property name="text">
                                    <string>All Applications</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QRadioButton" name="pendingRadioButton">
                                <property name="text">
                                    <string>Pending</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QRadioButton" name="approvedRadioButton">
                                <property name="text">
                                    <string>Approved</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QRadioButton" name="rejectedRadioButton">
                                <property name="text">
                                    <string>Rejected</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QTableWidget" name="leaveTable">
                    <property name="selectionMode">
                        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
                    </property>
                    <property name="sortingEnabled">
                        <bool>true</bool>
                    </property>
                    <attribute name="horizontalHeaderStretchLastSection">
                        <bool>true</bool>
                    </attribute>
                    <column>
                        <property name="text">
                            <string>Application ID</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>From Date</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>To Date</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Reason</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Status</string>
                        </property>
                    </column>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="buttonFrame">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <item>
                            <spacer name="horizontalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="closeButton">
                                <property name="text">
                                    <string>Close</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
